R/uvhydrograph-render.R

```diff
@@ -327,7 +335,13 @@ createSecondaryPlot <- function(uvInfo, secondarySeriesList,
   
   invertPlot <- secondarySeriesList[['inverted']]
   
-  lims <- calculateLims(secondarySeriesList[['corrected']][['points']])
+  if(!isEmptyOrBlank(secondarySeriesList[['corrected']][['points']][['value']])){
+    lims <- calculateLims(secondarySeriesList[['corrected']][['points']])
+    corrEstPoints <- secondarySeriesList[['corrected']][['points']][['value']]
```

Comment:
You're right, I keep conflating "estimated" and "corrected", which is incorrect.